discordbot.py

```diff
@@ -26,43 +26,311 @@ class DiscordEFTBot(LogicEFTBot):
     def bot_price(self, ctx: CommandContext, data: str) -> Union[str, discord.Embed]:
         log.info("%s - searching for %s (new)\n", ctx.channel, data)
         lang = self.db.get_lang(ctx.channel)
-        info = EFT.check_price(lang, data)
-        response = localized_string(
-            lang,
-            "price",
-            info.name,
-            info.price,
-            info.updated.strftime("%m/%d/%Y %H:%M"),
-        )
-        embed = discord.Embed(
-            title="LogicEFTBot",
-            url="https://eft.bot",
-            description="The Free Tarkov Bot",
-            color=0x780A81,
-        )
-        embed.set_thumbnail(url=info.img)
-        embed.add_field(name=info.name, value=response, inline=True)
-        return embed
-    
+        try:
+            price = EFT.check_price(lang, data)
+            embed = discord.Embed(
+                title=price.name,
+                url=price.wikiLink,
+                color=0x780A81,
+            )
+            embed.set_thumbnail(url=price.img)
+            embed.add_field(
+                name=localized_string(lang, "marketPrice"),
+                value=format(int(price.price), ","),
+                inline=True,
+            )
+            embed.add_field(
+                name=localized_string(lang, "marketTrader"),
+                value=price.traderName,
+                inline=True,
+            )
+            embed.add_field(
+                name=localized_string(lang, "marketTraderPrice"),
+                value=format(int(price.traderPrice), ","),
+                inline=True,
+            )
+            embed.add_field(
+                name=localized_string(lang, "marketSlot"),
+                value=format(round((price.price / price.slots)), ","),
+                inline=True,
+            )
+            embed.add_field(
+                name=localized_string(lang, "market7dAvg"),
+                value=format(int(price.avg7daysPrice), ","),
+                inline=True,
+            )
+            embed.add_field(
+                name=localized_string(lang, "market24hAvg"),
+                value=format(int(price.avg24hPrice), ","),
+                inline=True,
+            )
+            embed.set_footer(
+                text=localized_string(lang, "marketUpdated")
+                + maya.MayaDT.from_datetime(price.updated).slang_time()
+            )
+            return embed
+        except:

```

Comment:
two general notes:

1. try to limit the number of things you have inside your try: catch:
2. Try to avoid `except: ` as often as possible. You should be catching a specific type of exception if you can, and not catching _all_ exceptions if you don't mean it.

you write `except:` here, but you really mean, for instance, `except SomeNetworkException as e:`